frontend/src/routes/__layout@root.svelte

```diff
@@ -79,63 +56,7 @@
 		workspacePickerOpen = false
 	}
 
-	async function loadUserInfo() {
-		refreshSuperadmin()
-
-		if (!$usersWorkspaceStore) {
-			try {
-				usersWorkspaceStore.set(await WorkspaceService.listUserWorkspaces())
-			} catch {}
-		}
-		if ($usersWorkspaceStore) {
-			if (!$workspaceStore) {
-				workspaceStore.set(localStorage.getItem('workspace')?.toString())
-			}
-			if ($workspaceStore && $usernameStore) {
-				await getUser($workspaceStore)
-			} else if ($superadmin) {
-				console.log('You are a superadmin, you can go wherever you please')
```

Comment:
I think this logic has been removed? The reason for this is the ability for a superadmin to go any workspace (using the list as superadmin in the worskpace pages) and hence being in a workspace he has possible no derivable usernames of. Could you test if that is still possible and work well ?

Reply:
This behavior is working again as intended.